Makefile

```diff
@@ -45,7 +45,7 @@ kube-delete-database:
 
 kube-delete-all: kube-delete kube-delete-database
 
-kube-create:
+kube-create: update-manifests
```

Comment:
At this point (getting ready to release v2alpha) I don't mind how individual projects handle the kube abstractions in makefiles. I'll deal with that in a more definitive way on v2alpha-2 (storing this logic in a single place instead of trying to create a standard in 5 places).